src/inspect_ai/_view/www/src/app/log-list/grid/LogListGrid.module.css

```diff
@@ -93,6 +94,20 @@
   transition: background-color 0.15s ease-in-out;
   width: fit-content;
   min-width: 100%;
+  cursor: pointer;
+}
+
+.bodyRow:hover {
+  background-color: var(--bs-secondary-bg-subtle);
+}
+
+.selectedRow {
+  background-color: var(--bs-primary-bg-subtle);
+  border-left: 3px solid var(--bs-primary);
```

Comment:
This causes the layout to shift when a row is selected- let's either remove this apply some other css (e.g. a negative -3px margin or padding somewhere) to keep the layout stable.